### ISSUE TITLE
aspell failed on solaris

### ISSUE THREAD

**Issue by jessekps** [r-hub/rhub#339]
I am testing my package on rhub solaris and get the following error:
```
#> * this is package ‘dexter’ version ‘1.0.4’
#> * checking CRAN incoming feasibility ...Error in aspell(files, filter = list("dcf", ignore = ignore), control = control, :
#> Running aspell failed with diagnostics:
#> Error: /export/home/XHUNFqP/Rtemp/RtmpfTa4hy/aspell_personal303f54f12ac4: The word "UTF-8" is invalid. The character '-' may not appear at the middle of a word.
#> Execution halted
```
I removed the encoding specifier from the description file but the error remains. The only place where the word UTF-8 appears in any file in the package is in the vignetteencoding field in the vignettes and I'm pretty sure that's a correct usage, also the filter "dcf" should relate to DESCRIPTION file I believe. 

Am I doing something wrong or is there a problem with the R installation?

**Comment by gaborcsardi** [comment]
IDK, but I don't think you need to run the spell check on Solaris. Run it on Windows and Linux, that's enough.

**Comment by jessekps** [comment]
I did not intentionally run it. I guess it's included in the --as-cran option. I'll run it wothou, thanks

**Comment by vspinu** [comment]
Can the aspell be disabled on solaris? The errors are outright ridiculous:

```
   Error: The word "UTF-8" is invalid. The character '-' may not appear at the middle of a word.
```

**Comment by gaborcsardi** [comment]
AFAIK CRAN never runs with `--as-cran` on Solaris, and if you do, you can just ignore this error.

It would be somewhat confusing to have `check_for_cran()` set up a number of options, _except on Solaris_.

**Comment by vspinu** [comment]
yes, I could ignore but the execution stops on this error. Haven't tried but maybe `--no-stop-on-test-error` should be passed on.

**Comment by gaborcsardi** [comment]
Oh, right, that's not good, in that case we should fix this.

Btw. is there a particular reason you are running `--as-cran` on Solaris?

**Comment by gaborcsardi** [comment]
Btw.2. you can probably explicitly specify 
```
...
  check_args = "--as-cran",
  env_vars = c("_R_CHECK_CRAN_INCOMING_USE_ASPELL_" = "false")
...
```
to work around this

**Comment by vspinu** [comment]
:) good question.

 I just run `rhub::check_for_cran(...)` through ESS devtools shortcut. It allows me to start the check interactively for whatever build I want and I find it very convenient. Of course this issue could be worked-around in the ESS interface, if it's solaris, run with `rhub::check`, otherwise `rhub::check_for_cran`. But, from the UI prospective, it doesn't seem warranted to add two devtools shortcuts on the ESS side, one for check, one for check_for_cran.

**Comment by vspinu** [comment]
> Btw.2. you can probably explicitly specify

Just did and there is another silly error which interrupts the check process:

```
1894#> This system is set to the wrong time: please correct

1895#> system: 2020-03-14 11:50 (UTC)

1896#> correct: 2020-03-14 10:50 (UTC)

1897#> * DONE

1898#> Status: 1 ERROR, 1 NOTE

1899#> See

1900#> ‘/export/home/X03s9M2/lubridate.Rcheck/00check.log’

1901#> for details.
```

https://builder.r-hub.io/status/lubridate_1.7.8.tar.gz-4258181a79934f6e9cc8bcf218cea531

**Comment by gaborcsardi** [comment]
This issue is about the previous R-hub system, it does not apply to the new system, so I am closing it now. Please see https://r-hub.github.io/rhub/ for the new system, R-hub v2.
